src/constants.js

```diff
@@ -24,8 +24,15 @@ const fontSizes = [
   72
 ]
 
+const measures = [
+  24,
+  32,
+  48
```

Comment:
In the next major version, I plan on changing the em-unit breakpoints to be pixel values to keep everything consistent. Would be great to handle this with pixels as well, but still allow em-units to be defined with a string